assets/prototype/domain/shared/predicates/dateUtilities/datesAreEqual.ts

```diff
@@ -0,0 +1,7 @@
+import { isEqual } from 'date-fns';
+
+const datesAreEqual = (dateLeft: Date | number, dateRight: Date | number): boolean => {
```

Comment:
I was thinking that since this function lives under `dateUtilities` we might simplify it's name by a similar approach as is `date-fns`
![image](https://user-images.githubusercontent.com/1477453/70985079-38645600-20c4-11ea-9fac-8cb41f343536.png)

Reply:
smth like `areEqual` or `isEqual`

Reply:
i like that idea... in this case `areEqual` makes the most sense imho
despite it running the risk of making you sound like a pirate when you say the name

Reply:
Updated.

Reply:
Agree, pronunciation is a tricky domain: https://www.youtube.com/watch?v=Dd7FixvoKBw

Reply:
> Video unavailable
> The uploader has not made this video available in your country.

😢

Reply:
That's unacceptable.
Here is the query: https://www.youtube.com/results?search_query=Substitute+Teacher